src/SignalRServiceExtension/Bindings/SignalRIAsyncCollectorExtensions.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Azure.SignalR.Management;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+
+namespace Microsoft.Azure.WebJobs
+{
+    public static class SignalRIAsyncCollectorExtensions
```

Comment:
Extra `I`?

Reply:
I don't think so. This class contains extension methods for `IAsyncCollector`.

Reply:
Oh, you consider it extension of `IAsyncCollector` instead of `SignalRAsyncCollector`. Would it be extend SignalRAsyncCollector more accurate?